tests/app/test_jinja_filters.py

```diff
@@ -237,12 +237,18 @@ def test_format_duration(duration, formatted_duration, app):
 @pytest.mark.parametrize(
     "answer, width",
     (
-        ({}, 10),
+        ({}, 15),
         ({"maximum": {"value": 1}}, 1),
         ({"maximum": {"value": 123456}}, 6),
         ({"maximum": {"value": 12345678901}}, 15),
         ({"minimum": {"value": -123456}, "maximum": {"value": 1234}}, 7),
         ({"decimal_places": 2, "maximum": {"value": 123456}}, 8),
+        ({"maximum": {"value": 999_999_999_999_999}}, 15),
+        ({"decimal_places": 5, "maximum": {"value": 999_999_999_999_999}}, 20),
+        ({"decimal_places": 6, "maximum": {"value": 999_999_999_999_999}}, 30),
+        ({"minimum": {"value": -99_999_999_999_999}}, 15),
+        ({"decimal_places": 5, "minimum": {"value": 999_999_999_999_999}}, 20),
```

Comment:
I think these last two were meant to be negative, currently they duplicate the two on lines 247 & 248, otherwise looks good